contract-map.json

```diff
@@ -1976,5 +1976,19 @@
     "erc20": true,
     "symbol": "NEXO",
     "decimals": 18
+  },
+  "0xE9E73E1aE76D17A16cC53E3e87a9a7dA78834d37": {
```

Comment:
Please remove also CAMP from JSON in this place because it is duplication and resolve conflicts against master

Reply:
I've processed your confirmation.